java/test/jmri/jmrit/audio/JavaSoundAudioSourceTest.java

```diff
@@ -37,5 +37,12 @@ public void setUp() {
     }
 
     @After
-    public void tearDown() {        JUnitUtil.tearDown();    }
+    public void tearDown() {
+        // this created an audio manager, clean that up
+        InstanceManager.getDefault(jmri.AudioManager.class).cleanup();
+
+        jmri.util.JUnitAppender.suppressErrorMessage("Unhandled audio format type 0");
+        jmri.util.JUnitAppender.suppressWarnMessage("Initialised Null audio system - no sounds will be available.");
```

Comment:
Don't think we need this as NullAudioFactory no longer emits this as a WARN message, rather as INFO